wazo_amid/plugins/config/http.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2015-2023 The Wazo Authors  (see the AUTHORS file)
```

Comment:
This copyright is wrong (copy/paste). Should be 2024 only

Reply:
https://github.com/wazo-platform/wazo-amid/pull/89